docs/UsingNetlifyGuide.md

```diff
@@ -0,0 +1,25 @@
+## Using RepoSense with Netlify
+
+[Netlify](https://www.netlify.com) enables you to run RepoSense with your configured files without downloading the latest jar file.
```

Comment:
This sound like the whole objective is to avoid one download. The reader would wonder, why so much work to gain so little?

Reply:
Prefer to remove it or clarify further why doing this would be beneficial?

Reply:
Clarify further.